spec/lib/export_dsi_approver_records_to_big_query_spec.rb

```diff
@@ -43,7 +43,7 @@
       'given_name' => Faker::Name.first_name,
       'family_name' => Faker::Name.last_name,
       'email' => Faker::Internet.email,
-      'organisation' => { 'urn' => 100_000 }
+      'organisation' => { 'urn' => 100_000, 'uid' => 100_000 }
```

Comment:
Please make the values different for the different keys so that the test can fail when the wrong one is returned.